### ISSUE TITLE
Causes Problems with PHP SESSIONS, think it sends twice [code.google.com]

### ISSUE THREAD

**Issue by fedosov** [fedosov/webug#6]
Reported by mathewrssll, Feb 20, 2011

**What steps will reproduce the problem?**
1. Created two php functions, one to generate a form token hash and another to check if hidden form token hash field matches the $_SESSION token.
1. It works in every browser including chrome but with this addon enabled it causes problems as outlined below. 

What is the expected output? the php function to check if form token matches session token comes back saying it's invalid. This addon seems to alter the token or sends data twice causing the session token and form token to be a mismatch. A Majour bug.

What do you see instead?

What version of the product are you using? Latest Version

On what operating system? Windows 7
# 

This addon causes session problems with PHP. I am a web developer and i had a problem i was trying to figure out for days. 

I created two php functions one to generate a form token hash and to echo out in a hidden field in a form and the other function was to see if the the session token matched the hidden form token. 

After many hours of debugging and getting help from a pro developer we realised the problem was only happening in Chrome. We disabled every addon and Chrome was working fine, we then renabled every addon one by one and found that Webug was causing the issue when enabled. 

What happens is when i load the webpage that has the hidden form token on it and compare the hidden form token in form to the session token they never match and are different (even after submitting the form). I tried numerous methods in my function to overcome the issue and got a pro in and it seems this addon somehow alters the token value so the session token value never matches the token in the form either that or this addon submits the data twice or something. I see another issue someone else created about submitting data twice so maybe they are linked together somehow.

Problem is i created a sample form with the functions etc on it and it works. This addon seems to cause confliction with my code somewhere. 
# 

So in simple terms on my website with addon enabled the session token wont match the hidden form field token but if i disable addon it will. 

But if i create a sample form with the two functions and the form and the form processing code and try it with addon enabled i don't get any problems. It seems the addon is conflicting with other code on my site but i cannot understand what it must be conflicting with. My code is valid and works in any other browser without problems. I just need to know what this addon is doing to cause it to conflict with my code.
